scoring.js

```diff
@@ -43,39 +43,39 @@
         const rFum = receiving.fumbles * -3
         const kReturns = player.stats.return.kickreturn
         const pReturns = player.stats.return.puntreturn
-        const krYards = kReturns.yards / 10
+        const krYards = kReturns.yards / 15
         const krTD = kReturns.touchdowns * 6
         const krFum = kReturns.fumbles * -3
-        const prYards = pReturns.yards / 10
+        const prYards = pReturns.yards / 15
         const prTD = pReturns.touchdowns * 6
         const prFum = pReturns.fumbles * -3
-        return sum [Yards + TD + Fum + rec + rYards + rtouchdowns + rFum + krYards + krTD + krFum + prYards + prTD + prFum]
+        return  Yards + TD + Fum + rec + rYards + rtouchdowns + rFum + krYards + krTD + krFum + prYards + prTD + prFum
     }
     const teStats = (player) => {
         const receiving = player.stats.receiving
         const rec = receiving.receptions * 1
         const rYards = receiving.yards / 10
         const rtouchdowns = receiving.touchdowns * 6
         const rFum = receiving.fumbles * -3
-        return sum [rec + rYards + rtouchdowns + rFum]
+        return rec + rYards + rtouchdowns + rFum
 
 
     }
     
     
     const calculateScore = (player) => {
         if (player.position === 'QB'){
-            return qbStats
-    
+            return qbStats(player)
```

Comment:
Remember with long if else statements it makes more sense and is more efficient to use a switch statement.  Think of this the way that you did the web calculator.